cmd/forwarder/fixer.go

```diff
@@ -56,6 +63,13 @@ func fix(r io.Reader, remoteAddr string, logplexDrainToken string) ([]byte, erro
 			messageWriter.Write(b)
 		}
 
+		if authUser != "" {
+			messageWriter.WriteString(" log_iss_user=" + authUser)
```

Comment:
If we're really concerned about efficiency here, it's probably better to invoke `messageWriter.WriteString` twice rather than concatenating two strings to create a new string here.

Reply:
I was thinking the same thing. However, @freeformz commented earlier to opt for this method, so I deferred to that. =D